### PR TITLE
Issue 1 sql query endpoint for python

### PR DESCRIPTION
**Pull request by dionboles-asym**
#1  I added a static method to Q named sql  this method sends raw sql to the sql endpoint in the spring service

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, ubuntu-latest, windows-latest]
-        python-version: [3.6, 3.9, 3.9.5]
+        python-version: [3.6, 3.9, 3.9.6]
```
Comment (nemanjab17) [inline .github/workflows/tests.yml:11]: Why are 3.7 and 3.8 skipped?

File: cdapython.py [hunk 2]
```diff
@@ -63,6 +65,17 @@ def __init__(self, *args) -> None:
...
         self.query.r = _r
+       
+    @staticmethod
+    def sql(sql, host=CDA_API_URL, dry_run=False, offset=0, limit=1000):
+        with ApiClient(
+            configuration= Configuration(host=host)
+        ) as api_client:
+            api_instance = QueryApi(api_client)
+            api_response = api_instance.sql_query(sql)
+        if dry_run is True:
```
Comment (nemanjab17) [inline cdapython.py:76]: Could be replaced with `if dry_run:`

Comment (dionboles-asym) [reply cdapython.py:76]: You are right about that thanks i did miss that

File: .github/workflows/tests.yml (hunk 1 above)
Comment (dionboles-asym) [reply .github/workflows/tests.yml:11]: We should be testing for those verisons. I'm currenly currently using the python version i have installed and that github actions supports. so i will add the support verisons thanks for this

**Pull request merged**